CreeDictionary/CreeDictionary/settings.py

```diff
@@ -271,52 +267,21 @@
         "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
     )
 
-log_dir = Path(BASE_DIR) / "django_logs"
-log_dir.mkdir(exist_ok=True)
-
```

Comment:
Normally I really like having persistent logs on disk, but wow is

sapir:/data/texts/opt/cree-intelligent-dictionary/CreeDictionary/django_logs/django.log

ever noisy with all sorts of ValueError / AttributeError exceptions, and no
timestamps :s When running under systemd, console logs will at least be timestamped.

*note that systemd logs are by default cleared on reboot*

Reply:
Thanks! I redirected to console and made a new issue: #596